docs/developer/explanations/decisions/0002-switched-to-pip-skeleton.rst

```diff
@@ -0,0 +1,31 @@
+2. Started using the the dls python skeleton
+============================================
+
+Date: 2022-02-18
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+We should use the diamond light source `python3-pip-skeleton <https://github.com/DiamondLightSource/python3-pip-skeleton>`_.
+The skeleton is used across diamond to ensure consitency in developer environments and package management.
+
+Decision
+--------
+
+We have switched to using the skeleton.
+
+Consequences
+------------
+
+This module will use a fixed set of tools as developed in python3-pip-skeleton and can pull from this skeleton to update the packaging to the latest techniques.
```

Comment:
Nitpick: I always reflow text to 80 characters in RST files using https://marketplace.visualstudio.com/items?itemName=stkb.rewrap (Alt+Q)